worldedit-core/src/main/java/com/fastasyncworldedit/core/configuration/Settings.java

```diff
@@ -567,7 +567,8 @@ public FaweLimit getLimit(Actor actor) {
         public int DISCARD_AFTER_MS = 60000;
 
         @Comment({
-                "When using fastmode also do not bother to fix existing ticking blocks"
+                "When using fastmode also do not bother to fix existing ticking blocks",
+                "Only works in versions up to 1.17.2"
```

Comment:
1.17.2?

Reply:
1.18 will always call recalcBlockCounts as it's in the constructor for LevelChunkSection thus we can't "disable" it

Reply:
2. is wrong then nevertheless 👀

Reply:
Why?